tests/metrics/process/test_comm.py

```diff
@@ -0,0 +1,9 @@
+from pydriller.metrics.process.comm import COMM
+
+def test_zero_commits_count():
```

Comment:
Tests look good to me. You should add a test in which there is a renaming though